### PR TITLE
Slightly cleanup RendererBase docs.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

**Tests and Styling**
- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).

**Documentation**
- [ ] New features are documented, with examples if plot related.
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

File: lib/matplotlib/backend_bases.py [hunk 1]
```diff
@@ -321,18 +319,17 @@ def draw_gouraud_triangles(self, gc, triangles_array, colors_array,
     def _iter_collection_raw_paths(self, master_transform, paths,
                                    all_transforms):
         """
-        Helper method (along with :meth:`_iter_collection`) to implement
-        :meth:`draw_path_collection` in a space-efficient manner.
+        Helper method (along with `_iter_collection`) to implement
+        `draw_path_collection` in a space-efficient manner.
```
Comment (dstansby) [inline lib/matplotlib/backend_bases.py:323]:
```suggestion
        `draw_path_collection` in a memory-efficient manner.
```
Perhaps? Maybe "space-efficient" has a context-specific meaning that I don't know and is fine.

**Review by dstansby — APPROVED** [review]
Feel free to take or leave my comment, and then self merge

File: lib/matplotlib/backend_bases.py (hunk 1 above)
Comment (anntzer) [reply lib/matplotlib/backend_bases.py:323]: edited.

**Comment by anntzer** [comment]
Self-merging per @dstansby's comment.

**Pull request merged**